### PR TITLE
Clarify that shard allocations settings are not mutually exclusive

### PR DESCRIPTION
**Pull request by hlcianfagna**

### CODE REVIEW

**Review by amotl — COMMENTED** [review]
Will probably be good, but please confirm, @hammerhead.

File: docs/general/ddl/shard-allocation.rst [hunk 1]
```diff
@@ -38,6 +38,12 @@ defined by altering a table) from one set of nodes to another:
    Assign the table to a node whose *{attribute}* has **none** of the
    comma-separated values.
 
+.. NOTE::
+
+    These settings are not mutually exclusive. We can for instance have a
```
Comment (hammerhead) [inline docs/general/ddl/shard-allocation.rst:43]: I think we typically don't use "we" in the documentation, but rather address the user as "you".

**Comment by hlcianfagna** [comment]
I also tested "zoneA" with camel case like that and it works.

**Review by hammerhead — APPROVED** [review]

**Comment by amotl** [comment]
retest this please

**Pull request merged**